conduct/COC_PROCESS_FOR_INCIDENT_MANAGEMENT.md

```diff
@@ -1,7 +1,17 @@
+This document defines how the OpenJS Foundation implements its code of conduct.
+
+If you want to report an incident, go to [code-of-conduct.openjsf.org][CoC].
+
+***
+
+The [Code of Conduct Team (CoC Team)](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/conduct/COC_POLICY.md#code-of-conduct-team-coc-team) is responsible for the maintenance of these processes.
+
+**These processes haven't been updated since the creation of the CoC Team. It is expected that the CoC Team will overhaul them in the near future.**
```

Comment:
Extremely large -1 against this. The CoC team should not be self-governing, and setting it up as such is extremely dangerous.

Reply:
More specifically, the CoC team should be _implementing_ rules and structures that the CPC sets up for them, asking for clarification on interpretations as needed. This way, there is no question about whether or not the actions of the CoC team to be questioned in the decisions they've made - they are just enacting what the CPC set up. It's shielding for both the CPC and the team.

Self-governance is something that's _actively_ avoided by the Node.js Moderation Team for the above reason. Also we've seen at least one other Foundation that's had pretty explosive issues with self-governance - justified or not.

Reply:
Would setting up a ratifying process for the CPC alleviate your concerns?

Reply:
Repeating this comment here on suggestion from @tobie -

From the content proposed in this branch, there is still some work to do on [this particular file](https://github.com/openjs-foundation/cross-project-council/blob/coc/conduct/COC_TEAM_CHARTER.md) but the groundwork is there for eligibility. For this item, I think we could come to closure more efficiently by landing this PR and opening a specific issue (and tracking it on the CPC calls).

It does appear that the CoC Team composition and operations is a specific area that requires additional work. I propose we open this as a separate and specific issue so as not to block the entire CoC policy. That issue can be highlighted on our regular CPC calls-

Reply:
Fixed in d6010efa4394e309d9ea68510e639f4371965f0d by leaving the responsibility of defining and maintaining the enforcement processes with the CPC. Of course, this doesn't mean that the CoC Team members cannot contribute to those, just that they remain under the responsibility of the CPC.

Reply:
@bnb can you please check this fix and resolve this conversation if you're happy with the fix?